tools/myst/host/strace.c

```diff
@@ -91,13 +91,73 @@ int myst_set_strace_filter(
     return 0;
 }
 
+int myst_set_strace_id_filter(
+    const char* id_filter,
+    myst_strace_config_t* strace_config,
+    bool tid)
```

Comment:
Could you add a bit comments here to explain if `tid` is true, set filter for tid, else its for pid?

Reply:
Sure, added